extensions/enrichedScatterplots.py

```diff
@@ -63,12 +64,30 @@ def main():
     x = [np.mean([float(negD[sn][pn]) for sn in negNames]) for pn in peptideNames]
     
     # Read in lists of enriched peptides
-    enrFiles = glob.glob("%s/*%s" % (opts.enrDir, opts.enrExt))
+    if opts.enrDir:
```

Comment:
Please switch the order here so that IF both '-e' and '-f' are provided, '-f' will be the default.